### ISSUE TITLE
Hard Fork Error

### ISSUE THREAD

**Issue by biddy87** [ethereum/mist#1031]
#### System information
``````
Version: `0.0.0`
OS & Version: windows/linux/osx
Node type: `eth/geth(default)`
``````

After using https://slacknation.github.io/medium/12/12.html

Steps 1 - 3 work fine
Step 4, after much delay, it finished, it took about 12 hours 
Step 5, when entering the password, getting the error "the method personal_signAndSend Transaction does not exist/is not available" 

Basically, pw error....pw was leveraged for other transactions including step 4 fine, please advise.

**Comment by luclu** [comment]
@biddy87 could you kindly fill out the system information field?

**Comment by biddy87** [comment]
OS X Yosemite 
10.10.4
downloaded mist: mist-macosx-0.8-1.zip

**Comment by biddy87** [comment]
node type: eth

**Comment by luclu** [comment]
> node type: eth

Ah there we go. Make sure to use eth in at least version `1.3.0` as it is the first supporting the HF. Otherwise you will end up on the non-fork chain. You could also use geth as eth is not yet running smoothly: https://github.com/ethereum/mist/issues/834 (here are some problems of Mist+eth: https://github.com/ethereum/mist/issues/607). Mind that you would need to move your account's keyfiles over to geth.

Please check if you are on the HF chain: https://github.com/ethereum/mist/issues/1028

**Comment by snario** [comment]
I downloaded Mist yesterday 0.8.1 and started a new node. I am hitting this error when following these instructions: https://steemit.com/ethereum/@pauls/ethereum-fork-step-by-step-guide-to-safely-splitting-your-eth-etc

![screen shot 2016-07-29 at 11 01 36 am](https://cloud.githubusercontent.com/assets/1933029/17252769/e010f448-557b-11e6-9cb9-7fa27d8cb8dc.png)
![screen shot 2016-07-29 at 11 01 48 am](https://cloud.githubusercontent.com/assets/1933029/17252768/e010be10-557b-11e6-90b4-d159a1bac01b.png)

**Comment by frozeman** [comment]
What geth  version are you using?

**Comment by frozeman** [comment]
The lates might not work. We first have to update mist

**Comment by luclu** [comment]
This should be fixed in `0.8.2`.
